### ISSUE TITLE
Error triggered by trying to use web.titlesnarfer - "no module named supybot"

### ISSUE THREAD

**Issue by inhahe** [ProgVal/Limnoria#1113]
Error triggered by trying to use web.titlesnarfer - "no module named supybot" 

OS: Windows 7
Python: 2.7.9
Limnoria: Supybot is 0.83.4.1+limnoria 2015.04.29

(i tried using SpiffyTitles instead and got the same issue.)

Here's the traceback I get;

```
Exception in thread Thread #2 (for snarfing http://www.m-w.com):
Traceback (most recent call last):
  File "D:\python27\lib\threading.py", line 810, in __bootstrap_inner
    self.run()
  File "D:\python27\lib\site-packages\supybot\commands.py", line 163, in run
    super(UrlSnarfThread, self).run()
  File "D:\python27\lib\threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "D:\python27\lib\site-packages\supybot\commands.py", line 214, in doSnarf
    f(self, irc, msg, match, *L, **kwargs)
  File "D:\python27\lib\site-packages\supybot\plugins\Web\plugin.py", line 100, in newf
    pn=self.name(), cn=f.__name__)
  File "D:\python27\lib\site-packages\supybot\commands.py", line 118, in process
    p.start()
  File "D:\python27\lib\multiprocessing\process.py", line 130, in start
    self._popen = Popen(self)
  File "D:\python27\lib\multiprocessing\forking.py", line 277, in __init__
    dump(process_obj, to_child, HIGHEST_PROTOCOL)
  File "D:\python27\lib\multiprocessing\forking.py", line 199, in dump
    ForkingPickler(file, protocol).dump(obj)
  File "D:\python27\lib\pickle.py", line 224, in dump
    self.save(obj)
  File "D:\python27\lib\pickle.py", line 331, in save
    self.save_reduce(obj=obj, *rv)
  File "D:\python27\lib\pickle.py", line 419, in save_reduce
    save(state)
  File "D:\python27\lib\pickle.py", line 286, in save
    f(self, obj) # Call unbound method with explicit self
  File "D:\python27\lib\pickle.py", line 649, in save_dict
    self._batch_setitems(obj.iteritems())
  File "D:\python27\lib\pickle.py", line 681, in _batch_setitems
    save(v)
  File "D:\python27\lib\pickle.py", line 286, in save
    f(self, obj) # Call unbound method with explicit self
  File "D:\python27\lib\pickle.py", line 748, in save_global
    (obj, module, name))
PicklingError: Can't pickle <function newf at 0x02FAFB30>: it's not found as supybot.commands.newf

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "D:\python27\lib\multiprocessing\forking.py", line 380, in main
    prepare(preparation_data)
  File "D:\python27\lib\multiprocessing\forking.py", line 489, in prepare
    file, path_name, etc = imp.find_module(main_name, dirs)
ImportError: No module named supybot
```

**Comment by ProgVal** [comment]
Have you tried restarting your bot?

**Comment by inhahe** [comment]
Yes, several times. I always get the error.
I didn't used to get this error, I'm not sure what i changed. It may have started when I switched from plain SupyBot to Limnoria.

**Comment by inhahe** [comment]
Just installed virtualenv so I could have both straight supybot and limnoria. 
Verified that the bug happens in limnoria and not in straight supybot, with the configuration files being almost the same.

**Comment by ProgVal** [comment]
Yes, I knew that because the bug happens in a part of the code that I wrote as a protection against DoS

**Comment by ProgVal** [comment]
Fixed by 9c57199.
